en/docs/install-and-setup/setup/deployment/kubernetes-deployment.md

```diff
@@ -1,36 +0,0 @@
-# Deploying the Micro Integrator on Kubernetes
```

Comment:
Better to add a redirect to the update page `Deploy the Micro Integrator on Kubernetes using Helm resources`

Reply:
Also make sure we have removed these pages from the MkDocs navigation as well.